src/openassetio-core/include/openassetio/hostApi/Manager.hpp

```diff
@@ -1482,8 +1485,263 @@ class OPENASSETIO_CORE_EXPORT Manager final {
                            const trait::TraitSet& resultTraitSet = {});
 
   /**
-   * Queries entity references that are related to the input reference
-   * by the relationships defined by a set of traits and their
+   * Query for entity references that are related to the input
+   * reference by the relationship defined by a set of traits and
+   * their properties.
+   *
+   * See documentation for the <!--
+   * --> @ref getWithRelationship(const EntityReferences&, <!--
+   * --> const trait::TraitsDataPtr&, size_t, <!--
+   * --> access::RelationsAccess, const ContextConstPtr&, <!--
+   * --> const RelationshipQuerySuccessCallback&, <!--
+   * --> const BatchElementErrorCallback& errorCallback, <!--
+   * --> const trait::TraitSet&)
+   * "callback variation" for more details on relationship behaviour.
+   *
+   * Any errors that occur during resolution will be immediately thrown
+   * as an exception, either from the @ref manager plugin (for errors
+   * not specific to the entity reference) or as a
```

Comment:
[consider] Copy-paste issue, but perhaps valid regardless - "not specific to the relationship" is probably better, though,

Reply:
Yeah, in the next signature we say "Errors that are not specific to the entity **relationship** will be  thrown as an exception"

Reply:
https://github.com/OpenAssetIO/OpenAssetIO/pull/1308/commits/6b4c9ab73199b7a58baa487747a6f94945c95a55